tests/plugins/high_entropy_strings_test.py

```diff
@@ -108,6 +108,25 @@ def test_ignored_lines(self, content_to_format):
 
         assert len(results) == 0
 
+    def test_entropy_lower_limit(self):
+        with pytest.raises(ValueError):
+            Base64HighEntropyString(-1)
+
+    def test_entropy_upper_limit(self):
+        with pytest.raises(ValueError):
+            Base64HighEntropyString(15)
+
+
+class TestBase64HighEntropyStrings(HighEntropyStringsTest):
+
+    def setup(self):
+        super(TestBase64HighEntropyStrings, self).setup(
+            # Testing default limit, as suggested by truffleHog.
+            Base64HighEntropyString(4.5),
```

Comment:
Super nit: Might be more DRY if we make this a hard-coded value, and export it from `usage.py`